frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -455,7 +457,7 @@ export const createOrUpdateResources = async (
 
   verb === 'create' && requests.push(createWebhookSecret(formData, 'generic', dryRun));
 
-  const defaultAnnotations = getAppAnnotations(repository, ref);
+  const defaultAnnotations = getGitAnnotations(repository, ref);
```

Comment:
Ah, this is the actual change... everything else was refactors. Took me a couple moments but I understand how this feature works 😄

Reply:
The actual change was just to remove `openshift.io/generated-by: OpenshiftWebConsole` this annotation from knservice but wanted to refactor some code along with that :stuck_out_tongue:

Reply:
Definitely a good example of where refactoring can be done along side a fix to not be noise but in fact the better solution 😄